lib/std/enumutils.nim

```diff
@@ -96,6 +97,12 @@ iterator items*[T: HoleyEnum](E: typedesc[T]): T =
     assert B[float].toSeq == [B[float].b0, B[float].b1]
   for a in enumFullRange(E): yield a
```

Comment:
I don't see why not to use this instead:
```nim
for a in E.toSeq: yield a
```

Reply:
I don't think this works, `toSeq` uses `items`, so you'd get a recursive definition.